cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -219,19 +218,16 @@ public class KubeMasterEnvironment implements MasterEnvironment {
   // In memory state for holding configmap name. Used to delete this configmap upon master environment destroy.
   private String configMapName;
   private CoreV1Api coreV1Api;
-  private RbacAuthorizationV1Api rbacV1Api;
   private KubeMasterPathProvider kubeMasterPathProvider;
   private LocalFileProvider localFileProvider;
-  private final Gson gson;
   private boolean workloadIdentityEnabled;
   private String workloadIdentityPool;
   private long workloadIdentityServiceAccountTokenTTLSeconds;
   private String programCpuMultiplier;
   private String cdapInstallNamespace;
+  private KubeJobCleaner kubeJobCleaner;
 
-  public KubeMasterEnvironment() {
-    gson = new Gson();
-  }
+  public KubeMasterEnvironment() {}
```

Comment:
We can remove this too since Java always generate a no-op default constructor if the class doesn't have a constructor.